packages/syft/src/syft/serde/recursive.py

```diff
@@ -249,13 +249,14 @@ def rs_proto2object(proto: _DynamicStructBuilder) -> Any:
     kwargs = {}
 
     for attr_name, attr_bytes_list in zip(proto.fieldsName, proto.fieldsData):
-        attr_bytes = combine_bytes(attr_bytes_list)
-        attr_value = _deserialize(attr_bytes, from_bytes=True)
-        transforms = serde_overrides.get(attr_name, None)
-
-        if transforms is not None:
-            attr_value = transforms[1](attr_value)
-        kwargs[attr_name] = attr_value
+        if attr_name != "":
```

Comment:
Q: Do we by mistake serialise empty string attributes, when does this happen?

Reply:
hmm, I dont remember anymore, let me look into that